.travis.yml

```diff
@@ -196,6 +196,17 @@ matrix:
     - &extended-vm
       name: "events"
       env: NAME=events EVENTS=events
+      language: python
+      python: 3.7
+      install:
+        # Install gcc
+        - source_pkg gcc
+        - arm-none-eabi-gcc --version
+        # Install python modules
+        - python -m pip install --upgrade pip==18.1
+        - python -m pip install --upgrade setuptools==40.4.3
+        - pip install -r requirements.txt
+        - pip list --verbose
```

Comment:
How was the events test working previously without these changes?

Reply:
They were included as part of psa-autogen test. The events test failed when I removed them.